### DISCUSSION TITLE
各位大佬训练模型的时候这个问题怎么解决UnicodeDecodeError: 'utf-8' codec can't decode byte 0xd3 in position 0: invalid continuation byte

### DISCUSSION THREAD

**Discussion by asgeagafasf** [open-mmlab/mmclassification#1183] [Q&A] [answered]
![(KZ3 P OM~`5}6AHEVR 29](https://user-images.githubusercontent.com/117829535/200834158-2d9d955b-6eee-40c4-b088-9e747df52266.png)

**Comment by yingfhu** [comment, marked as answer]
可以升级一下mmcls和mmcv的版本，mmcv1.5.1 修复了这个问题，https://github.com/open-mmlab/mmcv/issues/1903
推荐直接使用1.x 分支
